test/j_manual_approval_transfer_manager.js

```diff
@@ -397,17 +397,19 @@ contract("ManualApprovalTransferManager", accounts => {
                 }
             );
             console.log(JSON.stringify(verified[0]));
-            assert.equal(verified[0], true);
+            assert.equal(verified[0], 0x51);
```

Comment:
nb (readability): may use enum as in https://github.com/PolymathNetwork/polymath-core/pull/690/files